our py3-dbus is way newer, and their check depends on the removed distutils


diff --git a/src/hamster/client.py b/src/hamster/client.py
index 8a0c297..ab3d3ba 100644
--- a/src/hamster/client.py
+++ b/src/hamster/client.py
@@ -27,3 +27,2 @@ import sys
 from calendar import timegm
-from distutils.version import LooseVersion
 from gi.repository import GObject as gobject
@@ -44,12 +43,2 @@ from hamster.lib import datetime as dt
 
-# bug fixed in dbus-python 1.2.14 (released on 2019-11-25)
-assert not (
-    sys.version_info >= (3, 8)
-    and LooseVersion(dbus.__version__) < LooseVersion("1.2.14")
-    ), """python3.8 changed str(<dbus integers>).
-       That broke hamster (https://github.com/projecthamster/hamster/issues/477).
-       Please upgrade to dbus-python >= 1.2.14.
-    """
-
-
 class Storage(gobject.GObject):
